frontend/src/Modal/Modal.jsx

```diff
@@ -5,6 +5,10 @@ import RecipeList from "../RecipeList/RecipeList.jsx";
 
 const Modal = (props) => {
     const { pickedRecipe, closeModal} = props;
```

Comment:
are you using `closeModal` in here? If not pls remove

Reply:
I'm using closeModal on line 16.